cics-bundle-maven-plugin/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -26,7 +26,8 @@
             <id>default</id>
             <phases>
               <compile>
-                com.ibm.cics:cics-bundle-maven-plugin:build
+                com.ibm.cics:cics-bundle-maven-plugin:build,
+                com.ibm.cics:cics-bundle-maven-plugin:package
```

Comment:
Shouldn't this be moved to the package phase instead of the compile phase?

Reply:
OK great - thanks. I need to get more familiar with the available phases...!